bovine-be/cema_bovine.sql

```diff
@@ -4,14 +4,17 @@ create table cema_bovine
 		constraint cema_bovine_pkey
 			primary key,
 	tagging_date timestamp default now(),
-	birth_date timestamp default now(),
 	tag varchar(20) not null,
 	description varchar(250),
-	category varchar(50) default 'Ternero'::character varying not null,
-	status varchar(50) default 'Mamando'::character varying not null
 	genre varchar(10) not null,
-	establishment_cuig varchar(50) not null
+	establishment_cuig varchar(50),
+	birth_date date default now(),
+	category varchar(50) default 'Ternero'::character varying not null,
+	status varchar(50) default 'Mamando'::character varying not null,
+	operation_id uuid
```

Comment:
Will previous ones be generated?

Reply:
no, they will be left null

Reply:
same goes for all bovines added without an operation